src/hooks/useRoom.ts

```diff
@@ -21,6 +22,8 @@ export const useRoom = ({ room, venueName }: UseRoomProps) => {
 
   const roomUrl = room?.url ?? "";
 
+  const { push: openUrlUsingRouter } = useHistory();
```

Comment:
Could've named it `customOpenRelativeUrl`, to save us from reading ` customOpenRelativeUrl: openUrlUsingRouter` 😆